### ISSUE TITLE
TypeError: Cannot read property '_id' of undefined ... at Remote.overwriteFileAsync

### ISSUE THREAD

**Issue by Wenzel** [cozy-labs/cozy-desktop#1126]
### Description
Cozy Drive application on Linux `CozyDrive-3.9.0-x86_64.AppImage` is constantly missing one of my files to sync.

I tried to quit and open the app multiple times, it won't sync a specific file.

### Steps to Reproduce

I don't know what lead to this bug.

### Expected behavior

The file should have been uploaded to my CozyCloud

### Actual behavior

The file is only present locally
![capture d ecran de 2018-08-13 01-43-34](https://user-images.githubusercontent.com/964610/44007267-e446c8f2-9e9a-11e8-8473-97b1b56a572a.png)
![capture d ecran de 2018-08-13 01-46-42](https://user-images.githubusercontent.com/964610/44007275-f7e1ab34-9e9a-11e8-8bf7-8ee5ec9ff918.png)

![capture d ecran de 2018-08-13 01-47-07](https://user-images.githubusercontent.com/964610/44007272-e85ff904-9e9a-11e8-94a7-36d22b6fbea6.png)

### Informations

* **reproductible**: I don't know
* **OS**: Fedora 28
* **version**: 3.9.0

### Log output
By digging into `~/cozy-desktop/logs.txt`
~~~JSON
{
  "name": "Cozy Desktop",
  "hostname": "localhost.localdomain",
  "pid": 29440,
  "component": "Sync",
  "level": 50,
  "path": "wiki/tiddlywiki.info",
  "err": {
    "message": "Cannot read property '_id' of undefined",
    "name": "TypeError",
    "stack": "TypeError: Cannot read property '_id' of undefined\n    at Remote.overwriteFileAsync (/tmp/.mount_CozyDrZfoqG6/app/resources/app.asar/core/remote/index.js:174:68)\n    at <anonymous>"
  },
  "change": {
    "id": "wiki/tiddlywiki.info",
    "changes": [
      {
        "rev": "26-8397a5c5b3b74a15b9c79c112233e789"
      }
    ],
    "doc": {
      "path": "wiki/tiddlywiki.info",
      "docType": "file",
      "md5sum": "h9oR0LVoD0ZG+qXX3X5SDw==",
      "ino": 4474071,
      "updated_at": "2018-08-12T22:41:55.768Z",
      "mime": "application/octet-stream",
      "class": "application",
      "size": 1626,
      "sides": {
        "local": 26,
        "remote": 4
      },
      "tags": [],
      "errors": 1,
      "_id": "wiki/tiddlywiki.info",
      "_rev": "26-8397a5c5b3b74a15b9c79c112233e789"
    },
    "seq": 3099
  },
  "msg": "Cannot read property '_id' of undefined",
  "time": "2018-08-12T22:43:23.777Z",
  "v": 0
}
{
  "name": "Cozy Desktop",
  "hostname": "localhost.localdomain",
  "pid": 29440,
  "component": "Sync",
  "level": 20,
  "path": "wiki/tiddlywiki.info",
  "seq": 3100,
  "doc": {
    "path": "wiki/tiddlywiki.info",
    "docType": "file",
    "md5sum": "h9oR0LVoD0ZG+qXX3X5SDw==",
    "ino": 4474071,
    "updated_at": "2018-08-12T22:41:55.768Z",
    "mime": "application/octet-stream",
    "class": "application",
    "size": 1626,
    "sides": {
      "local": 27,
      "remote": 4
    },
    "tags": [],
    "errors": 2,
    "_id": "wiki/tiddlywiki.info",
    "_rev": "27-c3fc0d22c33b4072bda0ce2f32f08fed"
  },
  "msg": "Applying change...",
  "time": "2018-08-12T22:43:23.986Z",
  "v": 0
}
~~~
Full [logs](https://gist.github.com/Wenzel/154851d1b2f7020f04fcba19f999e5fe)

Thanks for your help.

**Comment by Wenzel** [comment]
By moving the file out and back in the folder being watched, i was able to get it synced:
![capture d ecran de 2018-08-13 02-00-02](https://user-images.githubusercontent.com/964610/44007372-ba722d58-9e9c-11e8-8cf4-363469b8678a.png)


But this was definitely a bug.

**Comment by sebn** [comment]
Hello there! Thank you for reporting this!
This is definitely a bug, we'll investigate soon.

**Comment by aenario** [comment]
Hello @Wenzel and sorry for the delay, this bug was caused by bad file metadata in the synchronisation database. We did not manage to find the root cause of the bug, but the last merged PR (https://github.com/cozy-labs/cozy-desktop/pull/1144) added a safety check before we save this kind of metadata into the database. 

If someone encounters the same root cause, it will give us an error with more context.

Thanks for the report !
